BotRoyaleUnity/Assets/Scripts/buildRobot.cs

```diff
@@ -7,57 +7,89 @@
 
 public class buildRobot : MonoBehaviour
 {
-    public GameObject block,center,spike;
-    public Vector3 pos,rot;
-    
-
-// Start is called before the first frame update
-void Start()
-    {   
-        string jsonString = File.ReadAllText("Assets/Scripts/test.json");
-        var N = JSON.Parse(jsonString);
-        string test = N[0]["type"];
-        int a = 0;
-        while(N[a] != null){
-            pos = new Vector3((float)N[a]["x"],0.5f,(float)N[a]["y"]);
-            string type = N[a]["type"];
-            Quaternion rot;
-            string direcction = N[a]["direction"];
-            switch(direcction){
-                case "north":
-                    rot = Quaternion.Euler(90,0,0);
-                    break;
-                case "south":
-                    rot = Quaternion.Euler(-90,0,0);
-                    break;
-                case "east":
-                    rot = Quaternion.Euler(0,0,-90);
-                    break;
-                case "west":
-                    rot = Quaternion.Euler(0,0,90);
-                    break;
-                default:
-                    rot = Quaternion.Euler(0,0,0);
-                    break;
+    public GameObject block, center, spike;
+
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        string jsonString = File.ReadAllText("Assets/Scripts/robot1.json");
```

Comment:
This code here that reads the json and builds the robot would be good to put into it's own method, because it's the same thing copied twice, and we'll likely be calling it from a network handler class

Reply:
When we add the websocket connection and I know how the JSON data will be received, I will make a method for that.